### DISCUSSION TITLE
Missing an example of mixed configured and not configured event

### DISCUSSION THREAD

**Discussion by imba-tjd** [community/community#25098] [Actions]
<p>In short, I’d like an example like this adding to <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows" rel="nofollow noopener">https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows</a>:</p>
<p><img src="https://user-images.githubusercontent.com/24759802/181088760-75cd5fcb-5dc3-4632-b160-045cc49a84eb.jpeg" alt="Annotation 2019-11-18 153125.jpg" data-base62-sha1="1MOTJ0UyIj4HR1CI3GDINxr7Alz" width="270" height="116"></p>
<p>pic: <a href="/t5/image/serverpage/image-id/2840iE2C08BFF5BC22967/image-size/large">https://github.community/t5/image/serverpage/image-id/2840iE2C08BFF5BC22967/image-size/large?v=1.0&amp;px=-1</a></p>
<hr>
<p>If I wrote:</p>
<p><img src="https://user-images.githubusercontent.com/24759802/181088761-24c40cef-a24d-4cd0-95ff-358f1f23c007.jpeg" alt="Annotation 2019-11-18 153213.jpg" data-base62-sha1="1Yw8iA6bKCxBGl4umeQaf1mmWgO" width="271" height="130"></p>
<p>pic: <a href="/t5/image/serverpage/image-id/2841i8361BD95CD9AA2CF/image-size/large">https://github.community/t5/image/serverpage/image-id/2841i8361BD95CD9AA2CF/image-size/large?v=1.0&amp;px=-1</a></p>
<p>I would get:</p>
<pre><code>yaml: line 5: mapping values are not allowed in this context
</code></pre>
<p>And the article says “Use an array when using more than one event”:</p>
<pre><code>on:[push,pull\_request]
</code></pre>
<p>Simply change it to YAML works:</p>
<p><img src="https://user-images.githubusercontent.com/24759802/181088762-3c64cc15-2cc6-4379-bc1e-f0b8bf0f6920.jpeg" alt="Annotation 2019-11-18 154214.jpg" data-base62-sha1="xaqTEZPlghm2DLFYJSQhuwrYi9Y" width="206" height="95"></p>
<p>pic: <a href="/t5/image/serverpage/image-id/2843i643591E1904AE2B0/image-size/large">https://github.community/t5/image/serverpage/image-id/2843i643591E1904AE2B0/image-size/large?v=1.0&amp;px=-1</a></p>
<p>But if you add more options, it fails again:</p>
<p><img src="https://user-images.githubusercontent.com/24759802/181088763-df03cbc6-8c7f-4ffc-97ea-3e4659f77e3b.jpeg" alt="Annotation 2019-11-18 153750.jpg" data-base62-sha1="scWWlZ2Zy6fTro5lxlxUWzWpwbU" width="322" height="207"></p>
<p>pic: <a href="/t5/image/serverpage/image-id/2844iBE895D711CE939C8/image-size/large">https://github.community/t5/image/serverpage/image-id/2844iBE895D711CE939C8/image-size/large?v=1.0&amp;px=-1</a></p>
<p>So I also suggest you adding some notes and detection to the online editor about this.</p>
<p>I didn’t learnt it from YAML syntax. I tried it out. It really puzzles me. And at this time I still don’t know what syntax it is.</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>When you want to trigger a workflow in multiple events with specified options, you can use next syntax:</p>
<p>on:</p>
<p>  pull_request:</p>
<p>    branches:</p>
<p>      - master</p>
<p>  push:</p>
<p>    branches:</p>
<p>      - master</p>
<p>  schedule:</p>
<p>    - cron: “0 0 * * *”</p>
<p>  issues:</p>
<p>     types: [opened, edited, milestoned]</p>
<p>If you don’t want to specific options for one of those events, you can just remove the options , leave : behind the event name.  </p>
<p>on:</p>
<p>  pull_request:</p>
<p>    branches:</p>
<p>      - master</p>
<p>  push:</p>
<p>  schedule:</p>
<p>    - cron: “0 0 * * *”</p>
<p>  issues:</p>
<p>     types: [opened, edited, milestoned]</p>
<p><img src="https://user-images.githubusercontent.com/24806478/181088765-a561f026-2425-4967-8b96-697314db0cff.png" alt="on multiple event.png" data-base62-sha1="2qshWkgdSU6T9z4JrmMYh6dBpB0" width="481" height="328"></p>

**Comment by imba-tjd** [comment]
<p>Thanks for your reply. I mainly request GitHub to add them to the documentation in case other people encounter the same issue.</p>

**Comment by Yanjingzhu** [comment]
<p>Sorry for misunderstanding your purpose of this ticket. I have submitted your suggestion (add mixed trigger events example in <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows" rel="nofollow noopener">document</a>) to appropriate engineering team . </p>
<p>Thank you for your contribution to Github Actions.  </p>

**Comment by timruffles** [comment]
<p>Thanks for the suggestion <a class="mention" href="/imba-tjd">@imba-tjd</a>!</p>

**Comment by Yanjingzhu** [comment]
<p>Good news!</p>
<p>We have added an example to the document:  <strong>using multiple events with activity types or configuration</strong>  </p>
<p><a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration" rel="nofollow noopener">https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration</a>  </p>
